src/core/reactor.cc

```diff
@@ -1442,7 +1441,7 @@ void cpu_stall_detector::generate_trace() {
     buf.append_decimal(uint64_t(delta / 1ms));
     buf.append(" ms");
     print_with_backtrace(buf, _config.oneline);
-    maybe_report_kernel_trace();
+    maybe_report_kernel_trace(buf);
```

Comment:
The patch description says:
> This will glue kernel trace to whatever raport a caller is generating and print all of it in a single blob to the log sink.

In fact it won't, because `print_with_backtrace()` would flush the buffer before it's passed down to maybe_report_kernel_trace, so printing will be anyway split into two parts. Next patch that moves flush to destructor really makes it happen